rudder-server-relay/SOURCES/rudder-apache-relay-ssl.conf

```diff
@@ -34,3 +34,35 @@ RewriteRule /policies/(.*?)/(.*) /var/rudder/share/$1/$2
     Include /opt/rudder/etc/rudder-networks-24.conf
   </IfVersion>  
 </Directory>
+
+
+<Location /rudder/relay-api/shared-folder>
+
+  <IfVersion < 2.4>
+    Order allow,deny
+    Include /opt/rudder/etc/rudder-networks.conf
+  </IfVersion>
+  <IfVersion >= 2.4>
+    Include /opt/rudder/etc/rudder-networks-24.conf
+  </IfVersion>
+
+</Location>
+
+RewriteCond %{REQUEST_METHOD}  =GET
+RewriteRule /rudder/relay-api/shared-folder/(.*) /var/rudder/configuration-repository/shared-files/$1
+
+<Directory /var/rudder/configuration-repository/shared-files>
```

Comment:
The location does not say the same thing, We need to define access to the directory through apache